File: 프로그래머스/0/181858. 무작위로 K개의 수 뽑기/무작위로 K개의 수 뽑기.py
def solution(arr, k):
    
    result = []
    for a in arr:
        if (a not in result and len(result) < k):
            result.append(a)
    
    if len(result) < k:
        while len(result) < k:
            result.append(-1)
    
    return result

File: 프로그래머스/0/181858. 무작위로 K개의 수 뽑기/무작위로 K개의 수 뽑기.java
import java.util.*;
class Solution {
    public int[] solution(int[] arr, int k) {
        List<Integer> result = new ArrayList<>();
        
        for(int a: arr){
            if(!result.contains(a) && result.size()<k){
                result.add(a);
            }
        }
        
        if(result.size() < k){
            while(result.size() < k){
                result.add(-1);
            }
        }

        return result.stream().mapToInt(Integer::intValue).toArray();
    }
}